typings/index.d.ts

```diff
@@ -98,12 +98,15 @@ declare module 'discord-player' {
     type FiltersStatuses = {
         [key in Filters]: boolean;
     }
+    type FiltersOption = {
+        [key in Filters]: boolean;
```

Comment:
This should probably be `[key in Filters]: string | boolean` since we want to allow custom filters. I think that's what @Androz2091 was referring to in [this comment](https://github.com/Androz2091/discord-player/pull/290/files#r579765736)

Reply:
I don't see any case where a filter would be a boolean?

Reply:
You have to set a boolean for existing filters? 

```ts
client.player.setFilters(message, {
	 bassboost: true, // existing filter
	"custom-filter": "custom-filter-value"
})
```

Reply:
So what do you want?